decidim-admin/app/assets/stylesheets/decidim/admin/extra/_user-groups_filter.scss

```diff
@@ -0,0 +1,50 @@
+.user-groups__dropmenu{
```

Comment:
Can you pr this to [design-admin](https://github.com/decidim/design-admin)? Maybe we can find a more general pattern for this? Feels a bit ad-hoc.